src/c++/perf_analyzer/test_report_writer.cc

```diff
@@ -90,4 +103,54 @@ TEST_CASE("testing WriteGpuMetrics")
   }
 }
 
+TEST_CASE("report_writer: WriteLlmMetrics")
+{
+  std::shared_ptr<ProfileDataCollector> collector;
+  CHECK_NOTHROW_MESSAGE(
+      pa::ProfileDataCollector::Create(&collector),
+      "failed to create profile data collector");
+
+  InferenceLoadMode infer_mode{10, 20.0};  // dummy values
```

Comment:
![gif](https://github.com/triton-inference-server/client/assets/7692737/e68ee8cb-851d-4c34-8a98-d3e6590f0ed6)

Reply:
nit: Can you leave out the values so it's more clear these numbers don't matter?

Reply:
Do you mean specify what these numbers mean like this?
```suggestion
  uint32_t concurrency{10};  // dummy value
  double request_rate{20.0};  // dummy value
  InferenceLoadMode infer_mode{concurrency, request_rate};
```

Reply:
```suggestion
  InferenceLoadMode infer_mode{};
```
Would this work?